app/views/shared/check_answers/_no_link_items.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-grid-row" id="app-check-your-answers__<%= name %>">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m"><%= question %></h2>
+  </div>
+  <% if answers.present? %>
+    <div class="govuk-grid-column-one-third govuk-summary-list--no-border align-change-link-right">
+      <p>
+        <%= link_to(url, class: 'govuk-link') do %>
+          <%= t('generic.change') %><span class="govuk-visually-hidden"><%= question %></span>
+        <% end %>
+      </p>
+    </div>
+  <% end %>
+</div>
+
+<dl class="govuk-summary-list govuk-!-margin-bottom-9">
+  <% if answers.present? %>
+    <% answers.each_with_index do |answer, index| %>
+      <%= content_tag(:div, class: 'govuk-summary-list__row normal-word-break', id: "app-check-your-answers__#{name}_#{index}") do %>
+        <div class="govuk-summary-list__value check-your-answers-question">
+          <%= answer %>
+        </div>
+        <div class="govuk-summary-list__value check-your-answers-answer">
+            <%= t('generic.yes') %>
+        </div>
+      <% end %>
+    <% end %>
```

Comment:
Can you use `check_answer_no_link` here? Something like:
```erb
<% answers.each_with_index do |answer, index| %>
   <%= check_answer_no_link(
              name: "{name}_#{index}",
              question: answer,
              answer: t('generic.yes')
           ) %>
  <% end %>
```

Reply:
Done!